docs/user-guide/configuration/workflow.md

```diff
@@ -215,6 +217,61 @@ jobs:
 
 Example repo: https://github.com/screwdriver-cd-test/workflow-remote-requires-example
 
+## Remote Join
+You can also have remote join jobs. Please double check with your cluster admin whether it is supported. 
+
+#### Example
+In the following example, this pipeline 3 will start the `join_job` job after successful completion of: internal_fork, external_fork in pipeline 2, _and_ external_fork in pipeline 4.
+
+![Remote join](../assets/remote-join.png)
+
+Pipeline 3 screwdriver.yaml:
+
+```
+shared:
+  image: node:12
+  steps:
+    - echo: echo hi
+jobs:
+  main:
+    requires: [~commit, ~pr]
+  internal_fork:
+    requires: [main]
+  join_job:
+    requires: [internal_fork, sd@2:external_fork, sd@4:external_fork]
+```
+
+Pipeline 2 screwdriver.yaml:
+
+```
+shared:
+  image: node:12
+  steps:
+    - echo: echo hi
+jobs:
+  external_fork:
+    requires: [~sd@3:main]
+```
+
+Pipeline 4 screwdriver.yaml:
+
+```
+shared:
+  image: node:12
+  steps:
+    - echo: echo hi
+jobs:
+  external_fork:
+    requires: [~sd@3:main]
```

Comment:
Question:

> Assuming all job names are valid and exist.

I'm confused in the `Caveats` section regards to `~`. For the name conventions, is it possible to have 

1)
```
requires: [~sd@3:main, ~sd@3:main2]
```
Does `~` here mean 1 `OR` for two jobs, or 1 AND for 2 `external require`d jobs? 

2) 
```
requires: [sd@3:main, sd@3:main2]
```

This means `AND` two jobs, right? (as before)

3) 

```
requires: [sd@3:main, ~sd@3:main2]
```

Does this mean `sd@3:main` is no longer to be considered as `external require`, while `~sd@3:main2` is considered as `external require`d job? 
Is this format considered as 1 `AND` for two jobs (`sd@3:main`, `~sd@3:main2`), or `OR` for 2 jobs. 

@tkyi @jithine

Reply:
tilde always represents `OR`; we're not changing syntax
just trying to say when the start node is external trigger(s), we do not support having`AND` logic for them

I'll change the wording of the caveat to try to make this more clear.